meshroom/ui/qml/main.qml

```diff
@@ -475,15 +478,22 @@ ApplicationWindow {
 
     // Utility functions for elements in the menubar
 
-    function initFileDialogFolder(dialog) {
-        if(_reconstruction.graph && _reconstruction.graph.filepath) {
-            dialog.folder = Filepath.stringToUrl(Filepath.dirname(_reconstruction.graph.filepath));
+    function initFileDialogFolder(dialog, importImages = false) {
+        let folder = "";
+        if (_reconstruction.graph && _reconstruction.graph.filepath) {
+            folder = Filepath.stringToUrl(Filepath.dirname(_reconstruction.graph.filepath));
         } else {
             var projects = MeshroomApp.recentProjectFiles;
             if (projects.length > 0 && Filepath.exists(projects[0])) {
-                dialog.folder = Filepath.stringToUrl(Filepath.dirname(projects[0]));
+                folder = Filepath.stringToUrl(Filepath.dirname(projects[0]));
             }
         }
+
+        if (importImages) {
+            folder = imagesFolder === "" ? folder : imagesFolder
```

Comment:
Should check with `Filepath.exists` or something similar instead to ensure the dialog folder is properly initialized

Reply:
Done.